### PR TITLE
add unit tests for safe cast

### PR DESCRIPTION
**Pull request by dianakocsis**
## Related Issue
adding more tests

## Description of changes
unit tests for SafeCast

### CODE REVIEW

File: test/libraries/SafeCast.t.sol [hunk 1]
```diff
@@ -33,12 +50,26 @@ contract SafeCastTest is Test {
...
+    function test_fuzz_ToInt128FromUint256(uint256 x) public {
         if (x <= uint128(type(int128).max)) {
             assertEq(uint128(SafeCast.toInt128(x)), x);
         } else {
             vm.expectRevert(SafeCast.SafeCastOverflow.selector);
             SafeCast.toInt128(x);
         }
     }
+
+    function testToInt128FromUint256() public {
```
Comment (snreynolds) [inline test/libraries/SafeCast.t.sol:69]: just a slight change to the format for the above tests

`test_toInt256`

**Review by snreynolds — COMMENTED** [review]
smol comment!

File: test/libraries/SafeCast.t.sol [hunk 2]
```diff
@@ -3,10 +3,10 @@ pragma solidity ^0.8.20;
 
 import {Test} from "forge-std/Test.sol";
 import {Vm} from "forge-std/Vm.sol";
-import {SafeCast} from "src/libraries/SafeCast.sol";
+import {SafeCast} from "../../src/libraries/SafeCast.sol";
 
 contract SafeCastTest is Test {
-    function testToUint160(uint256 x) public {
+    function test_fuzz_toUint160(uint256 x) public {
```
Comment (hensha256) [inline test/libraries/SafeCast.t.sol:9]: the others say like `_fromUint256`

Comment (dianakocsis) [reply test/libraries/SafeCast.t.sol:9]: oh thats just cuz theres 2 `toInt128`s, so i differentiated them by adding the `from`s. 
For `toUint160` and `toUint256` I didn't add the `from`s, but I can if it makes it easier

**Review by hensha256 — APPROVED** [review]

**Pull request merged**